Components/AccordionVertical/style.styl

```diff
@@ -0,0 +1,69 @@
+$rupture.scale = lookup('$globalRuptureScale') || (0 575px 767px 991px 1199px)
+$rupture.anti-overlap = lookup('$globalRuptureAntiOverlap') || -1px
+$containerPadding = lookup('$globalContainerPadding') || 15px
+$containerMaxWidth = lookup('$globalContainerMaxWidth') || 1140px
+$colorBrand = lookup('$globalColorBrand') || #254fc7
+$buttonBackgroundColor = #f3f3f3
+$itemPadding = 15px
+$iconHeight = 11px
+$iconWidth = 13px
+
+.flyntComponent[is='flynt-accordion-vertical']
+  *,
+  *:before,
+  *:after
+    box-sizing: border-box
+
+  .accordionVertical
+    center($containerMaxWidth, $containerPadding)
+
+    &-list
+      list-style: none
+      margin: 0
+      padding: 0
+
+    &-content
+      height: 0
+      overflow: hidden
+      padding: 0 $itemPadding
+      transition: height .3s ease
+
+      &-isSizing
+        transition: none
+
+    &-btn
+      -webkit-appearance: none
+      background-color: $buttonBackgroundColor
+      border: 0
+      cursor: pointer
+      display: block
+      font-size: 16px
+      outline: 0
+      padding: $itemPadding ($itemPadding + $iconWidth) $itemPadding $itemPadding
```

Comment:
Yeah this is to make sure that the icon does not overlap with the text if a long title is provided.